ClientApp/src/Api/apiClient.ts

```diff
@@ -55,15 +55,39 @@ export async function submitSighting(newSighting: NewSighting) {
   }
 }
 
+export interface RecentSightingResponse {
```

Comment:
I think this interface is the same as `SightingResponse` which is the same as `Sighting` in this file - could we move the 2 extra interfaces and just use one please?